omim2obo/utils/mondo_non_disease_exclusions/mondo_non_disease_exclusions.py

```diff
@@ -0,0 +1,43 @@
+"""Generates a Mondo intesional exclusions TSV file with non-diseases as its contents."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+
+def gen_mondo_non_disease_exclusions(symbolic_prefixes_path: str, mim_titles_path: str, outpath: str) -> pd.DataFrame:
+    """Generates a Mondo intesional exclusions TSV file with non-diseases as its contents."""
+    # Read inputs
+    symbols_df = pd.read_csv(symbolic_prefixes_path, sep='\t').fillna('')
+    mims_header = ['Prefix', 'MIM Number', 'Preferred Title; symbol', 'Alternative Title(s); symbol(s)',
+                   'Included Title(s); symbols']
+    mims_df = pd.read_csv(mim_titles_path, sep='\t', comment='#', header=None, names=mims_header).fillna('')
+    excludable_symbol_names = set(symbols_df[~symbols_df['can_be_disease_or_phenotype']]['name'])
+
+    # Transform
+    df = mims_df[mims_df['Prefix'].isin(excludable_symbol_names)][['MIM Number', 'Preferred Title; symbol']]\
+        .rename(columns={'MIM Number': 'term_id', 'Preferred Title; symbol': 'term_label'}).sort_values('term_id')
+    df['term_id'] = df['term_id'].apply(lambda x: f'OMIM:{x}')
+    df['exclusion_reason'] = 'MONDO:excludeNonDisease'
+    # todo: consider FALSE; will increase performance, and TRUE is just here as a failsafe.
+    df['exclude_children'] = 'TRUE'
+
+    # Save & return
+    df.to_csv(outpath, sep='\t', index=False)
```

Comment:
1/2 of the work to address: 
- https://github.com/monarch-initiative/mondo-ingest/issues/110

The other 1/2 will be done in `mondo-ingest`.

Output: https://github.com/monarch-initiative/omim/releases/download/2023-02-01/mondo_non_disease_exclusions.tsv

Header & example row
```tsv
term_id	term_label	exclusion_reason	exclude_children
OMIM:100640	ALDEHYDE DEHYDROGENASE 1 FAMILY, MEMBER A1; ALDH1A1	MONDO:excludeNonDisease	TRUE
```

I considered outputting something w/ 3 fields, `term_id`, `term_label`, and `non_disease_or_phenotype` (bool), and then including all OMIM terms in this TSV. But I figured it'd be simpler/better to export in the same format as our exclusion tables, with just the terms to be excluded.